api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -307,6 +307,26 @@
 			referencedTableName="users" referencedColumnNames="user_id"/>
 	</changeSet>
 	
+	<changeSet id="TRUNK-2497-Store-outgoing-MessageService-emails-in-a-table" author="ivange94">
+		<preConditions onFail="MARK_RAN">
+			<not><tableExists tableName="sent_message"/></not>
+		</preConditions>
+		<comment>
+			Create table to persist all sent messages
+		</comment>
+		<createTable tableName="sent_message">
+			<column name="message_id" type="int">
+				<constraints primaryKey="true" nullable="false"/>
+			</column>
+			<column name="sender_id" type="int"/>
+			<column name="receiver_id" type="int"/>
+			<column name="subject" type="varchar(50)"/>
+			<column name="content" type="varchar(50)"/>
```

Comment:
@dkayiwa nope. will fix that, until now i always varchar could take an abiturary number of characters. :), i guess i was not concentrating well in class when it was thought in first year or my teacher just thought me wrong.